example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -504,21 +503,50 @@ describe('The S3 Ingest Granules workflow', () => {
         expect(doesExist).toEqual(true);
       });
 
+      it('rejects moving a granule to a location that already exists', async () => {
+        const file = granule.files[0];
+
+        await s3().copyObject({
+          Bucket: config.bucket,
+          CopySource: `${file.bucket}/${file.filepath}`,
+          Key: `${testDataFolder}/${file.filepath}`
+        }).promise();
+
+        const destinations = [{
+          regex: '.*.hdf$',
+          bucket: config.bucket,
+          filepath: `${testDataFolder}/${file.filepath.substring(0, file.filepath.lastIndexOf('/'))}`
+        }];
+
+        const moveGranuleResponse = await granulesApiTestUtils.moveGranule({
+          prefix: config.stackName,
+          granuleId: inputPayload.granules[0].granuleId,
+          destinations
+        });
+
+        const responseBody = JSON.parse(moveGranuleResponse.body);
+
+        expect(moveGranuleResponse.statusCode).toEqual(409);
+        expect(responseBody.message).toEqual(
+          `Cannot move granule because the following files would be overwritten at the destination location: ${granule.files[0].name}. Delete the existing files or reingest the source files.`
+        );
```

Comment:
Do you mean, `reingest the source files` will put the files to the new locations?

Reply:
Can we add a `moveGranule` successful test case?  e.g. remove the existing file and try the `moveGranule` operation again.